corehq/apps/builds/models.py

```diff
@@ -132,6 +133,17 @@ def get_build(cls, version, build_number=None, latest=False):
     def all_builds(cls):
         return cls.view('builds/all', include_docs=True, reduce=False)
 
+    @classmethod
+    @quickcache([], timeout=5 * 60) #This seems to be not working.
```

Comment:
I could not see it caching the value for 5 mins. It was querying couchDB for all builds. So was not sure if i implemented it correctly. Can you help understand this?